packages/fluentui/react-northstar/src/components/Chat/chatContext.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext } from '@fluentui/react-bindings';
+
+export type ChatContextValue = {
+  compact: boolean;
+};
+
+export const ChatContext = createContext<ChatContextValue>({ compact: false });
```

Comment:
Using context selectors for single values might not always be necessary. In this case you could just use react native context and store a string value called `layout` and that can alternate between `compact` and `comfy`. The context could be called smth like `ChatLayoutContext`. There's no need to memoize a string primitive.

I understand this also might have the effect of changing the `compact` prop to something more like a `layout` prop which actually could be more extensible in the future.